CONTRIBUTING.md

```diff
@@ -45,4 +45,67 @@ on how to build iceberg locally.
 
 For Java styling, check out the section
 [Setting up IDE and Code Style](https://iceberg.apache.org/community/#setting-up-ide-and-code-style) from the
-documentation site. For Python, please use the tox command `tox -e format` to apply autoformatting to the project.
+documentation site.
+
+For Python, please use the tox command `tox -e format` to apply autoformatting to the project.
+
+### Java style guidelines
+
+#### Line breaks
+
+Continuation indents are 2 indents (= 4 spaces) from the start of the previous line.
```

Comment:
nit: Is `=` needed here? Could just be me but the first time I read it I thought of an IDE setting where you define `indent=2` when this is really just saying "2 indents, in other words 4 spaces".  I think "2 indents (4 spaces)" would read a little better.

Reply:
Fixed.